README.md

```diff
@@ -17,8 +28,60 @@ For more information on Fixtures, check out this [pytest documentation](https://
 A **Mock** is an object that simulates the behavior of another object such as a class or function. They are used to simulate the behavior of real objects for testing purposes.
 For more information on Mocks, check out this [unittest documentation](https://docs.python.org/3/library/unittest.mock.html) on the mock module.
 
-**Example:**
+## Installation
+
+To use the **JupyterHub Pytest Plugin**, you will first need to install it using pip:
+
+**Note**: Currently, Pytest JupyterHub can only be installed from GitHub since it is yet to be released on PyPI.
+
+```bash
+pip install git+https://github.com/jupyterhub/pytest-jupyterhub.git@main
```

Comment:
This will start working as soon as a git tag has been pushed with #48 merged and credentials to push to a PyPI project are setup.

```suggestion
pip install pytest-jupyterhub
```